### PR TITLE
fix(api-client): address bar background

### PR DESCRIPTION
**Pull request by antlio**
this pr fixes the address bar background color that was not showing up anymore by safelisting tailwing background utilities:

⊢ before / after
image
image

### CODE REVIEW

**Comment by antlio** [comment]
this doesn't do

**Comment by hwkr** [comment]
Could we just add them to this file? https://github.com/scalar/scalar/blob/main/packages/api-client/src/components/AddressBar/httpStatusCodeColors.ts

**Comment by antlio** [comment]
> Could we just add them to this file? https://github.com/scalar/scalar/blob/main/packages/api-client/src/components/AddressBar/httpStatusCodeColors.ts

we could! but would the issue remain as we add utilities programatically?

**Comment by hwkr** [comment]
> > Could we just add them to this file? https://github.com/scalar/scalar/blob/main/packages/api-client/src/components/AddressBar/httpStatusCodeColors.ts
> 
> we could! but would the issue remain as we add utilities programatically? however it's not necessarily the same topic here is it?

Tailwind decides what classes to include in the CSS by parsing all the Vue and TS files and doing a simple string search for anything that looks like it might be a Tailwind class. I think because the classes are being imported from OAS Utils that's why they aren't working.

We could also add it to the content list for the API Client but I think it's reasonable to just move it into the repo.

https://github.com/scalar/scalar/blob/main/packages/api-client/tailwind.config.ts#L10

Cursor-2024-12-05-06-55-07@2x

**Comment by antlio** [comment]
@hwkr i updated the tw config content

**Review by hanspagel — APPROVED** [review]
Sick

File: packages/api-client/tailwind.config.ts [hunk 1]
```diff
@@ -56,4 +56,9 @@ export default {
     },
   },
   plugins: [headlessPlugin, desktopVariants, colorMix()],
+  safelist: [
+    {
+      pattern: /bg-(red|green|blue|orange|yellow|purple|c-2)/,
```
[the remark this answers is not in the document]
Comment (antlio) [reply packages/api-client/tailwind.config.ts]: this has been removed, favored instead the importation of the files to the config

File: packages/api-client/tailwind.config.ts [hunk 2]
```diff
@@ -7,7 +7,11 @@ import { desktopVariants } from './src/tailwind/desktop-variants'
 
 export default {
   presets: [scalarPreset],
-  content: ['./index.html', './src/**/*.{js,jsx,ts,tsx,vue}'],
+  content: [
+    './index.html',
+    './src/**/*.{js,jsx,ts,tsx,vue}',
+    '../oas-utils/src/**/*.{js,ts,vue}',
```
Comment (hwkr) [inline packages/api-client/tailwind.config.ts:13]: Could add a comment here like Hans mentioned :)

**Review by hwkr — APPROVED** [review]
Looks good!!

File: packages/api-client/tailwind.config.ts (hunk 2 above)
Comment (antlio) [reply packages/api-client/tailwind.config.ts:14]: ✅

**Pull request merged**